frontend/views/components/Graphs/Progress.vue

```diff
@@ -1,20 +1,33 @@
-<template>
-  <span class="c-progress">
-    <span class="c-primary" :style="{ width: primary }"></span>
-    <span class="c-secondary" :style="{ width: secondary }" v-if="secondary"></span>
-  </span>
+<template lang="pug">
+.c-progress
+  .c-primary(
+    :style='{ width: primary + "%" }'
+  )
+  .c-secondary(
+    v-if='secondary'
+    :style='{ width: secondary + "%" }'
+  )
```

Comment:
Why make this take up 7 lines instead of 2? I think it would look great being just 2 lines...

Reply:
I understand your feeling but this is the result of fews rules applied across the app:
- v-if is the first parameter (https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended)
- Only one parameters per line (help to order element according to first rule, prevent attributes duplication, improve readability and help separation between normal and dynamic attributes)
- Same presentation between components that look like each other or have the same functionality to help comparison

If you don't like one of this rule, I can remove it across the app for consistency.

Reply:
That sounds reasonable. Personally I prefer more compact, but if you prefer it it this way and have some sort of convention that you're following that's 👌